internal/pkg/v2/model/core/data/event.go

```diff
@@ -0,0 +1,18 @@
+//
```

Comment:
Please articulate in this review the difference between DTO and model so that it's clear whether the implementation direction aligns with the months of discussion related to V2. It's unclear that this difference is understood given where the above DTOs have been placed.

Reply:
The changes have been made to add the `Event` type struct to [Event DTO file ](https://github.com/edgexfoundry/edgex-go/blob/a1b7b6c8e8fae61bece8c24517cf07fa7b5af9d4/internal/pkg/v2/dto/core/data/event.go) as well based on the suggestions.

DTOs are for API request/response object definition, and data models under `internal/pkg/v2/model` are for internal handler and db connection usage. Hopefully the code structure align with the v2 API design, thanks.